admin-client/src/pages/organization/OrgsReport.svelte

```diff
@@ -20,6 +22,13 @@
       <td>
         {org.agency}
       </td>
+      <td>
+        {#if org.isSelfAuthorized }
+          { formatDistanceStrict(new Date(org.selfAuthorizedAt), new Date(), { addSuffix: true, roundingMethod: 'floor' }) }
```

Comment:
Do we care about a date just for self authorized? Should we just use the `isSelfAuthorized` boolean?

Reply:
Do we care about the date in general? Should this field just be stored as a boolean?

The reason I'm showing it is that I saw that it's shown in the admin orgs index page, so I thought I ought to be consistent and show it here. In the onscreen report I show boolean and date in one column, similar to how it's done in the orgs index page. 

In the CSV export if I were to only include one column in the CSV I would have thought it should be the date one, but right now I've got them in separate columns on the theory it might make downstream sort/filter semantics a small bit simpler.

Reply:
Reviewing it more, I think we only care if an org is or is not self authorized. Using `No` in conjunction with a date since for `Self Authorized` is mixing the types and the current Self Authorized field in the orgs table shows similar ambiguity with the timestamp and an `N/A`. If you want to change the org table's Self Authorized field to boolean too that would be great.

It should simplify the component and a future boolean filter on self authorization is more pertinent in targeting groups.

Reply:
All right, I will only show and export the boolean in these reports. I think changing the underlying field should happen on a separate issue and PR, though. I'll spin that up next.